tests/integration/targets/eos_pyateos/tests/eapi/pyateos.yaml

```diff
@@ -0,0 +1,138 @@
+---
+- debug: msg="START eapi/eos_pyateos.yaml on connection={{ ansible_connection }}"
+
+- name: imstall python3-pip
+  package:
+    name: python3-pip
+    state: latest
+  delegate_to: 127.0.0.1
+
+
+- name: install jsondiff,jmespath
+  delegate_to: 127.0.0.1
+  pip:
+    name: "{{ item }}"
+  loop:
+    - jsondiff
+    - jmespath
```

Comment:
This isn't needed, you can add them into https://github.com/ansible-collections/eos/blob/master/requirements.txt

Reply:
Thanks! I am still trying to get my way around the new set-up and tests :)